project-base/storefront/helpers/isServer.ts

```diff
@@ -1,12 +1,7 @@
 import { NextIncomingMessage } from 'next/dist/server/request-meta';
+import { canUseDom } from './canUseDom';
 
-export const isServer = (): boolean =>
-    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-    !(
-        typeof window !== 'undefined' &&
-        typeof window.document === 'object' &&
-        typeof window.document.createElement === 'function'
-    );
+export const isServer = (): boolean => !canUseDom();
```

Comment:
What about rename `canUseDom` to `isClient`. And then we can remove this `isServer` and use it like `!isClient`?

Reply:
Im good with that 😁 done